changelog.md

```diff
@@ -66,7 +66,8 @@
 
 - Added `nim --eval:cmd` to evaluate a command directly, see `nim --help`.
 
-
+- `cast` now works in VM in more contexts, behaving like in RT; eg: `int <=> ptr T | ref T`,
+  see `tcastint.testCastRefOrPtr`
```

Comment:
I do really appreciate your contributions but it does feel like not even you read your own diffs. So now people that read our release changelogs should dig into some test cases? Come on.

Reply:
removed reference to `tcastint.testCastRefOrPtr` which was there as optional further reference, but yes, this could've been a pointer to a paragraph in manual instead of testcase. (and yes, I read my diffs)